front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -295,6 +296,27 @@ class Sidebar extends PureComponent<
           onData={this.handleData}
         />
         <MenuInner id="sidebar">
+          <Box w="100%">
+            <Link to="/">
+              <Box
+                height={
+                  isPagesAndMenuPage ? `${stylingConsts.menuHeight}px` : '60px'
+                }
+                background="#7FBBCA" // TODO: Use color from component library.
```

Comment:
Using this color for consistency as it is the same used on the border color for selected menus but It would be nice if these were coming from the defined colors in the component library

Reply:
Maybe we could at least have one constant here in CL instead of copying it?